nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/impl/ListProperty.java

```diff
@@ -48,22 +48,21 @@
      */
     protected final Field field;
 
-    protected final List<Property> children;
+    protected final List<Property> children = new ArrayList<>();
 
     public ListProperty(Property parent, Field field) {
         super(parent);
         this.field = field;
-        children = new ArrayList<>();
     }
 
     public ListProperty(Property parent, Field field, int flags) {
         super(parent, flags);
         this.field = field;
-        children = new ArrayList<>();
     }
 
     @Override
     public void internalSetValue(Serializable value) throws PropertyException {
+        // ListProperty uses a specific setValue implementation
```

Comment:
Why not making it a facade that would call the specific setValue implementation rather than doing nothing ?

Reply:
I'm not changing the way it works here, I'm just adding a comment explaining why this method is empty.

Furthermore, on the generic implementation, `setValue` calls `internalSetValue` so the opposite is not expected.